src/Language/PureScript/Errors.hs

```diff
@@ -1190,11 +1190,13 @@ prettyPrintSingleError (PPEOptions codeColor full level showDocs relPath) e = fl
     printRow f t = markCodeBox $ indent $ f prettyDepth t
 
     -- If both rows are not empty, print them as diffs
-    printRows :: Type a -> Type a -> (Box.Box, Box.Box)
-    printRows r1@RCons{} r2@RCons{} = let
+    -- If verbose print all rows else only print unique rows
```

Comment:
Since `full` is already in scope here, I think it would be preferable to have `printRows` retain the old signature, i.e. not take a new `Bool` argument, and instead just switch on `full` in the implementation.

Reply:
I replaced the implementation with a `case` expression, and It's now using the `full` variable that's in the scope.